docs/technical-guide/getting-started/recommended-settings.md

```diff
@@ -10,3 +10,33 @@ Disk requirements depend on your usage, with the primary factors being database
 As a rule of thumb, start with a **minimum** database size of **50GB** to **100GB** with elastic sizing capability — this configuration should adequately support up to 10 editors. For environments with **more than 10 users**, we recommend adding approximately **5GB** of capacity per additional editor.
 
 Keep in mind that database size doesn't grow strictly proportionally with user count, as it depends heavily on how Penpot is used and the complexity of files created. Most organizations begin with this baseline and elastic sizing approach, then monitor usage patterns monthly until resource requirements stabilize.
+
+
+# About Valkey / Redis requirements
+
+Valkey is mainly used for coordinating websocket notifications and as a cache (especially since Penpot 2.11). Therefore, disk storage will not be necessary as it will use the instance's RAM.
```

Comment:
Proposal to avoid ambiguity:

"Valkey is mainly used for coordinating websocket notifications and, since Penpot 2.11, as a cache. Therefore..."